zero-thrs-control/src/thrs/control/controllers.py

```diff
@@ -49,92 +52,132 @@ def setpoint(self, value: SetpointUnit):
         self._pid.setpoint = value
 
     def __call__(self, measurement: SetpointUnit, time: datetime) -> ValueUnit:
+        if self._reset_time: #TODO: deal with dt is simulation versus reality. In simulation, passing dt = tick_duration works. In real time, having actual time would be nice..
+            self._reset_time = False
         self._pid.time_fn = lambda: time.timestamp()
         pid_result = cast(ValueUnit | None, self._pid(measurement))
-        return pid_result if pid_result is not None else self._initial
+        return (
+            pid_result if pid_result is not None else self._initial
+        )  # do we want initial back or None?
```

Comment:
`pid_result` is only `None` if it was never enabled, so then `initial` makes sense